gradio/utils.py

```diff
@@ -680,3 +680,28 @@ def validate_url(possible_url: str) -> bool:
 
 def is_update(val):
     return type(val) is dict and "update" in val.get("__type__", "")
+
+
+def mount_gradio_app(
```

Comment:
Nit/suggestion: it feels more natural to me for this method to belong in `routes.py` with all of the other fastapi-related stuff

Reply:
Done!